setlexsem/generate/generate_data.py

```diff
@@ -1,7 +1,9 @@
 import argparse
```

Comment:
These should all be considered functions for generating **sets**, no? **Data** is unhelpfully generic and misleading.

Reply:
yup, will rename these